backend/src/main/java/quarano/department/activation/ActivationCodeException.java

```diff
@@ -1,10 +1,12 @@
 package quarano.department.activation;
 
+import lombok.Getter;
 import quarano.department.activation.ActivationCode.ActivationCodeIdentifier;
 
 /**
  * @author Oliver Drotbohm
  */
+@Getter
```

Comment:
I agree. Ideally the exception could expose an `I18nedMessage` directly and keep all the `Status` evaluation internal.

Reply:
I have no idea how to integrate `I18nedMessage` here. Do you have an example or some documentation on this?

Reply:
On a second thought, we don't even need `I18nedMessage` here. The exception could simply return the message key based on the status it holds internally so that the key can be forwarded into the `rejectField(…)` method.